packages/frontend/src/components/DashboardFilter/Filter.tsx

```diff
@@ -1,447 +0,0 @@
-            return getConditionalRuleLabelFromItem(filterRule, field);
-        } else {
-            const column = Object.values(sqlChartTilesMetadata)
-                .flatMap((tileMetadata) => tileMetadata.columns)
-                .find(
-                    ({ reference }) => reference === filterRule.target.fieldId,
-                );
-            if (column) {
-                return getConditionalRuleLabel(
-                    filterRule,
-                    getFilterTypeFromItemType(column.type),
-                    column.reference,
-                );
-            }
-            return;
-        }
-    }, [filterRule, field, sqlChartTilesMetadata]);
-
-    const filterRuleTables = useMemo(() => {
-        if (!filterRule || !field || !allFilterableFields) return;
-
-        return getFilterRuleTables(filterRule, field, allFilterableFields);
-    }, [filterRule, field, allFilterableFields]);
-
-    const hasUnsetRequiredFilter =
-        filterRule?.required && !hasFilterValueSet(filterRule);
-
-    const inactiveFilterInfo = useMemo(() => {
-        if (
-            activeTabUuid &&
-            appliesToTabs &&
-            !appliesToTabs.includes(activeTabUuid)
-        ) {
-            const appliedTabList = appliesToTabs
-                .map((tabId) => {
-                    return `'${
-                        dashboardTabs.find((tab) => tab.uuid === tabId)?.name
-                    }'`;
-                })
-                .join(', ');
-            return appliedTabList
-                ? `This filter only applies to tab${
-                      appliesToTabs.length === 1 ? '' : 's'
-                  }: ${appliedTabList}`
-                : 'This filter is not currently applied to any tabs';
-        }
-    }, [activeTabUuid, appliesToTabs, dashboardTabs]);
-
-    const handleClose = useCallback(() => {
-        if (isPopoverOpen) onPopoverClose();
-        closeSubPopover();
-    }, [isPopoverOpen, onPopoverClose, closeSubPopover]);
-
-    const handleSaveChanges = useCallback(
-        (newRule: DashboardFilterRule) => {
-            if (isCreatingNew && onSave) {
-                onSave(newRule);
-            } else if (onUpdate) {
-                onUpdate(newRule);
-            }
-            handleClose();
-        },
-        [isCreatingNew, onSave, onUpdate, handleClose],
-    );
-
-    return (
-        <>
-            <Popover
-                position="bottom-start"
-                trapFocus
-                opened={isPopoverOpen}
-                closeOnEscape={!isSubPopoverOpen}
-                closeOnClickOutside={!isSubPopoverOpen}
-                onClose={handleClose}
-                disabled={disabled}
-                transitionProps={{ transition: 'pop-top-left' }}
-                withArrow
-                shadow="md"
-                offset={1}
-                arrowOffset={14}
-                withinPortal
-            >
-                <Popover.Target>
-                    {isCreatingNew ? (
```

Comment:
Created both components from this one. The main split was this condition. 
From here, I deleted the components and then the unused properties and hooks.